packages/node-app/src/controllers/giveaway.controller.ts

```diff
@@ -136,14 +136,15 @@ export const updateGiveaway = async (req: Request, res: Response) => {
   const file = req.file as Express.Multer.File;
   try {
     const { id } = req.params;
-    const { title, description, prize, rules } = req.body;
+    const { title, description, prize, rules, manual } = req.body;
     const image = file ? fileToBase64(file as Express.Multer.File) : undefined;
     const updateFields = getDefinedFields({
       title,
       description,
       prize,
       rules,
       image,
+      manual
```

Comment:
Do we want to allow manual flag updates?

Reply:
Currently the manual flag doesn't add any functionally so there is no warm in having that already in the code. In the future, even if giveaways are marked as non-manual there will still be the possibility to trigger them manually just in case something goes wrong with the scheduler, so this is just a flag that tell us if we should trigger the scheduler to generate winners or not once that part is working.